controller/src/command.rs

```diff
@@ -713,6 +714,26 @@ pub struct TxsArgs {
 	pub tx_slate_id: Option<Uuid>,
 }
 
+/// Initial preparation of the wallet for migration
+pub fn migration<L, C, K>(
```

Comment:
Can you be more specific on the migration function name? Like `lmdb_sqlite_migration`